src/emerger.sh

```diff
@@ -3,6 +3,12 @@
 SRC=$(dirname "$(readlink -f "$0")")
 source $SRC/utils/global.sh
 
+ARGV=$@
+if [[ $@ =~ "-help" ]]; then
+    cat $SRC/utils/help
```

Comment:
What ```$@ =~``` stands for?

Reply:
`If argument values contain`

Reply:
> `If argument values contain`

Ok so ```$@``` stands for the array of arguments you pass after ```up```

Reply:
Indeed

Reply:
> Indeed

Okok, I'm gonna merge this changes